src/iuse_actor.cpp

```diff
@@ -1168,6 +1168,9 @@ void inscribe_actor::load( JsonObject &obj )
         material_whitelist.push_back("silver");
     }
 
+    verb = obj.get_string( "verb", _("Carve") );
+    gerund = obj.get_string( "gerund", _("Carved") );
```

Comment:
The loaded string is never translated, `verb.c_str()` needs to be wrapped with `_()`. The line here loads and stores the untranslated string from JSON or stores the translated default (if the JSON doesn't have that member).

The strings from JSON is not extracted from translation either. This can be easily fixed: add entries for "verb" and "gerund" to the `use_action_msgs` array in "lang/extract_json_strings.py".